torch/csrc/jit/passes/quantization.h

```diff
@@ -61,7 +61,8 @@ TORCH_API void InsertObservers(
     const std::string& method_name,
     const std::unordered_map<
         std::string,
-        std::tuple<script::Module, script::Module>>& qconfig_dict);
+        std::tuple<script::Module, script::Module>>& qconfig_dict,
+    bool inplace = false);
```

Comment:
The function needs to return the cloned module. Also, please update the comment correspondingly.